Description: Enables the compilation of the JMX classes and tweaks the Javadoc.
Author: Thomas Koch <thomas@koch.ro>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -167,12 +167,12 @@
         <and>
             <available classname="javax.management.MBeanInfo"
                        classpath="${jmx.jar}"/>
-            <available classname="com.sun.jdmk.comm.HtmlAdaptorServer">
+            <!--available classname="com.sun.jdmk.comm.HtmlAdaptorServer">
                 <classpath>
                     <pathelement location="${jmx.jar}"/>
 					<pathelement location="${javaee-api.jar}"/>
                 </classpath>
-            </available>
+            </available-->
         </and>
     </condition>
   </target>
@@ -497,7 +497,6 @@
              protected="true"
              author="true"
              use="true"
-             overview="${docs.dest}/overview.html"
              doctitle="log4j version ${version}&lt;br&gt;API Specification"
              windowtitle="Log4j Version ${version}"
              header='&lt;b&gt;Log4j ${version}&lt;/b&gt;&lt;!--  Licensed to the Apache Software Foundation (ASF) under one or more
@@ -517,8 +516,6 @@
 --&gt;'
              bottom="Copyright 2000-2007 Apache Software Foundation.">
 
-      <link href="http://java.sun.com/j2se/1.3/docs/api/"/>
-      <link href="http://java.sun.com/j2ee/sdk_1.3/techdocs/api/"/>
       <classpath refid="compile.classpath"/>
     </javadoc>
 
